src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -1,22 +1,67 @@
-.dropdown-filters .mat-form-field {
+.filters .mat-form-field {
   margin: 8px;
   font-size: 14px;
   max-width: 20%;
   width: 14%; /* depends on number of filters*/
 }
 
-.search-bar {
-  width: 80%;
+ul {
+  margin: 8px;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  list-style-type: none;
 }
 
-.dropdown-filters {
+.search-bar {
   width: 100%;
 }
 
+.filters {
+  width: 57%;
+  display: flex;
+}
+
 .label-filter-grid-tile {
   width: 100%;
 }
 
-.submit-button {
-  margin-left: 8px;
+.menu {
+  padding: 0 10px;
+  border-left: 1px solid #eee;
+  display: none;
+  cursor: pointer;
+
+  @media (max-width: 1148px) {
+    display: block;
+    justify-items: right;
+  }
+}
+
+.filters {
+  display: flex;
+  align-items: center;
+
+  @media (max-width: 1148px) {
+    display: none;
+
+    &.is-open {
+      display: block;
+      position: absolute;
+      width: 100%;
+      top: calc(20% + 8px); /* 8px margin from ul */
+      z-index: 10;
+      background-color: #fffcfc;
+
+      .filter-item {
+        display: inline-block;
+      }
+    }
+  }
+
+  .filter-item {
+    padding: 0 10px;
+    display: block;
```

Comment:
Is there a need for display: block and display: inline-block? Children of flexbox are automatically given display: inline-block